PokeAlarm/Manager.py

```diff
@@ -872,13 +901,12 @@ def process_egg(self, egg):
 
         time_str = get_time_as_str(egg['raid_end'], self.__timezone)
         start_time_str = get_time_as_str(egg['raid_begin'], self.__timezone)
-
-        gym_info = self.__gym_info.get(gym_id, {})
+        gym_info = self.get_gym_details(gym_id)
 
         egg.update({
-            "gym_name": gym_info.get('name', 'unknown'),
-            "gym_description": gym_info.get('description', 'unknown'),
-            "gym_url": gym_info.get('url', 'https://raw.githubusercontent.com/RocketMap/PokeAlarm/master/icons/gym_0.png'),
+            "gym_name": gym_info.get('name'),
```

Comment:
Why were the defaults removed?

Reply:
the defaults are set in the function that gets gym_info, in part because in the future we should not use github files as a source for these pictures. (much like the !pickle command in RM replaces the original files)

Reply:
What does the !pickle command in RM do?

Reply:
sorry, I meant to type `!sprite` - it's a custom discord command that gives you a download link to custom sprites to replace the original ones that are just numbers